generators/example/src/main/scala/Configs.scala

```diff
@@ -14,7 +16,7 @@ class BaseRocketConfig extends Config(
   new freechips.rocketchip.system.DefaultConfig)
 
 class DefaultRocketConfig extends Config(
```

Comment:
This  name seems highly confusing. Also why are we getting rid of Rocket only configs?

Reply:
Are you refering to having both the `Base` and the `Default` configs? Also, we are not getting rid of Rocket configs... what are you refering to?